README.md

```diff
@@ -32,22 +32,66 @@ As Administrator:
 
 ### Docker
 
-**NB** - `cht-conf` in a Docker container should only be used if you're familiar with running containers. You will have to copy or mount your config files into the container in order to be able to use the `cht` call to interact with a remote CHT instance.
+CHT Conf can also be run from within a Docker container. This is useful if you already have Docker installed and do not wish to configure the various dependencies required for developing CHT apps on your local machine. The necessary dependencies are pre-packaged in the Docker image.
 
-Build and then run the `cht-conf` container:
+#### Building the image
 
-	docker build -t cht-conf:v0 .
-	docker run -d --name cht-conf cht-conf:v0
+Run the following commands to create a new project directory and build the Docker image:
 
-You now have a container running in the background called `cht-conf`. You can get a shell on this container with:
+```shell
+mkdir cht-project
+cd cht-project
+curl https://raw.githubusercontent.com/medic/cht-conf/main/Dockerfile > Dockerfile
+docker build -t cht-conf .
+```
 
-	docker exec -it cht-conf /bin/bash
+#### Using the image
 
-And then you can run the utility with `cht`.  
+The resulting Docker image can be used as a [VSCode Development Container](https://code.visualstudio.com/docs/devcontainers/containers) (easiest) or as a standalone Docker utility.
 
-When you are done with the container, you can stop it with:
+##### VSCode Development Container
 
-	docker stop cht-conf
+If you want to develop CHT apps with VSCode, you can use the Docker image as a Development Container. This will allow you to use the `cht-conf` utility and its associated tech stack from within VSCode (without needing to install dependencies like NodeJS on your host system).
+
+[Install VSCode](https://code.visualstudio.com/) if you do not have it already.
+
+Run the following commands from within your project directory to download the `.devcontainer.json` config file, install the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers), and open the project directory in VSCode:
+
+```shell
+curl https://raw.githubusercontent.com/medic/cht-conf/main/devcontainer.config/.devcontainer.json > .devcontainer.json
+code --install-extension ms-vscode-remote.remote-containers
+code -n .
```

Comment:
Let's assume that if they have an existing project directory, they'll know how to use it.  But that if they don't, they'll need a lot of help, and so default to creating it for them. As well, let's quiet up `curl`. Finally let's call out they need to use the `wsl` shell not the `cmd` shell:

```suggestion
Using a WSL shell (Start -> wsl) to run the following commands. It will create a new project directory, download the `.devcontainer.json` config file, install the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers), and open the project directory in VSCode:

```shell
mkdir -p ~/cht-vscode-project && cd  ~/cht-vscode-project
curl -s https://raw.githubusercontent.com/medic/cht-conf/main/devcontainer.config/.devcontainer.json > .devcontainer.json
code --install-extension ms-vscode-remote.remote-containers
code -n .
```

Reply:
So, they should have already created the project directory above (when building the Docker image). Added a note here to clarify...

Also, I think it is a good idea to mention WSL for Windows, but I hesitate to make that the only language we use since anecdotally a lot of app developers use non-Windows systems too.

Reply:
ah - dude - why am I only thinking of WSL?!  That's silly of me and you're right - so many OSes out there!  your mention of WSL is great.  I'll resolve this.